src/docs/asciidoc/advancedFeatures.adoc

```diff
@@ -496,191 +499,195 @@ parallel-2 -> 8
+Since version `3.1.0`, Reactor comes with an advanced feature that is somewhat comparable
+to `ThreadLocal` but applied to a `Flux` or a `Mono` instead of a `Thread`: the `Context`.
 
 IMPORTANT: This is an advanced feature that is more targeted at library developers. It
-requires good understanding of the lifecycle of a `Subscription`, and is intended for
+requires good understanding of the lifecycle of a `Subscription` and is intended for
 libraries that are responsible for the subscriptions.
 
-A `Context` is an interface reminiscent of `Map`: it stores key-value pairs and allows you
-to fetch a value you stored by its key. More specifically:
-
-  * both key and values are `Object`, so a `Context` can contain any number of highly
-  different values from different libraries and sources.
-  * a `Context` is *immutable*.
-  * use `put(Object key, Object value)` to store a key-value pair, returning a new `Context`
-  instance. You can also merge two contexts into a new one using `putAll(Context)`.
-  * you can check if the key is present with `hasKey(Object key)`.
-  * use `getOrDefault(Object key, T defaultValue)` to retrieve a value (cast to a `T`) or
-  fall back to a default one if the Context doesn't have that key.
-  * use `getOrEmpty(Object key)` to get an `Optional<T>` (the context will attempt to cast
-  the stored value to `T`).
-  * use `delete(Object key)` to remove the value associated to a key, returning a new
-  `Context`.
-
-TIP: *Creating a* `Context`: You can create pre-valued contexts with up to five key-value
-pairs using the static `Context.of` methods. It takes 2, 4, 6, 8 or 10 `Object`, each
-couple of `Object` being a key-value pair to add to the `Context`. +
- +
-Alternatively you can also create a `Context.empty()`.
-
+A `Context` is an interface reminiscent of `Map`: it stores key-value pairs and lets you
+fetch a value you stored by its key. More specifically:
+
+* Both key and values are of type `Object`, so a `Context` can contain any number of
+highly divergent values from different libraries and sources.
+* A `Context` is *immutable*.
+* Use `put(Object key, Object value)` to store a key-value pair, returning a new
+`Context` instance. You can also merge two contexts into a new one by using
+`putAll(Context)`.
+* You can check if the key is present with `hasKey(Object key)`.
+* Use `getOrDefault(Object key, T defaultValue)` to retrieve a value (cast to a `T`) or
+fall back to a default one if the Context does not have that key.
+* Use `getOrEmpty(Object key)` to get an `Optional<T>` (the context attempts to cast the
+stored value to `T`).
+* Use `delete(Object key)` to remove the value associated to a key, returning a new
+`Context`.
+
+TIP: When *creating a* `Context`, you can create pre-valued contexts with up to five
+key-value pairs by using the static `Context.of` methods. They take 2, 4, 6, 8 or 10
+`Object` instances, each couple of `Object` instances being a key-value pair to add to
+the `Context`.
```

Comment:
reintroduce the ` +` ("space plus")  here and line below to keep the "alternatively ..." sentence within the TIP admonition